packages/sarif-to-comment/src/cli.ts

```diff
@@ -97,7 +101,8 @@ export function run() {
             sarifContentSourceRoot: cli.flags.sarifContentSourceRoot,
             ghActionAuthenticationMode: cli.flags.action,
             ruleDetails: cli.flags.ruleDetails,
-            suppressedResults: cli.flags.suppressedResults
+            suppressedResults: cli.flags.suppressedResults,
+            title: cli.flags.title
```

Comment:
📝 https://github.com/azu/security-alert/blob/5be23f660a2ff4af68317b34132e0853f65829ce/packages/sarif-to-comment/src/index.ts#L9
`title` is optional. Its valid.